pyproject.toml

```diff
@@ -118,7 +118,8 @@ testpaths = [
 ]
 markers = [
     "slow: marks tests as slow (deselect with '-m \"not slow\"')",
-    "gpu: marks tests that require a gpu (deselect with '-m \"not gpu\"')"
+    "gpu: marks tests that require a gpu (deselect with '-m \"not gpu\"')",
+    "mcmc: marks tests that require MCMC sampling (deselect with '-m \"not mcmc\"')"
```

Comment:
Great. How long do the MCMC tests run, currently?

Should we include them in the GitHub workflow (slow tests are disabled by default, if these are similary "slow" they should also be disabled, or additionally marked slow).

Reply:
without `-n auto` about 850s on my machine (this includes `slow`). with `-n auto` it's 4500s. I will need to investigate this further, but I can imagine that with multiple chains, we might cause more harm than good with test parallelization. this could interest you, too @Baschdl

Reply:
In this case we could think about running the mcmc tests sequentially (e.g. in CI) but a more sustainable solution might be to add a `pytest.mark` for tests that should only be executed sequentially. At this point, a legitimate question would be whether the parallelization of the tests brings enough benefit to outweigh the maintenance cost.